usim/_primitives/condition.py

```diff
@@ -10,14 +10,48 @@
 
 class Condition(Notification):
     """
-    A logical condition that triggers when ``True``
+    An asynchronous logical condition
+
+    Every :py:class:`~.Condition` can be used both in a
+    asynchronous *and* boolean context.
+    In an asynchronous context,
+    such as ``await``,
+    a :py:class:`~.Condition` triggers when :py:const:`True`.
```

Comment:
This reads strange at the moment. Maybe explicitly state "when the :py:class:`~.Condition` becomes :py:const:`True`".